boa/src/exec/tests.rs

```diff
@@ -385,6 +385,62 @@ fn unary_pre() {
     assert_eq!(&exec(execs_before_dec), "true");
 }
 
+#[test]
+fn unary_typeof() {
+    let string_type_result = String::from("string");
+    let number_type_result = String::from("number");
+    let undefined_type_result = String::from("undefined");
+    let boolean_type_result = String::from("boolean");
+    let object_type_result = String::from("object");
+    let function_type_result = String::from("function");
+    let symbol_type_result = String::from("symbol");
+    let typeof_string = r#"
+        const a = String();
+        typeof a;
+    "#;
+    assert_eq!(exec(typeof_string), string_type_result);
```

Comment:
I think this is more clear if we do it like this:
```suggestion
    assert_eq!(&exec(typeof_string), "string");

```
Also adding a new line after each assert, to separate them.